addons/website_event_exhibitor/models/event_sponsor.py

```diff
@@ -19,7 +19,8 @@ class EventSponsor(models.Model):
         'mail.thread',
         'mail.activity.mixin',
         'website.published.mixin',
-        'chat.room.mixin'
+        'chat.room.mixin',
+        'website.searchable.mixin'
```

Comment:
Maybe reorder to set the `'website.searchable.mixin'` under the `'website.published.mixin'` one ? :slightly_smiling_face: